rmgpy/solver/base.pyx

```diff
@@ -1399,4 +1438,13 @@ class TerminationRateRatio:
     
     def __init__(self, ratio=0.01):
         self.ratio = ratio
-        
+
+class TerminationAveMW:
+    """
+    Represent an average MW at which a simulation
+    should be terminated. This class has one attribute the average
+    molecular weight.  Useful for molecular growth situations.
+    """
+
+    def __init__(self, aveMW=50.0):
+        self.aveMW = aveMW
```

Comment:
Also `avgMW` here (and everywhere else it's used).

Could you add a note in the docstring about the units?

Reply:
done